docs/reference/helm-config.txt

```diff
@@ -136,12 +136,27 @@ Helm, the deployment should be configured by editing the ``values.yaml`` and
     during distributed training. A valid port range is in the format of
     ``MIN:MAX``.
 
+  - ``forcePullImage``: Defines the default policy for forcibly pulling images from
+    the docker registry and bypassing the docker cache. If a pull policy is specified
+    in the :ref:`experiment config <exp-environment-image>` this default  is overriden.
+    Defaults to ``false``.
+
   - ``cpuPodSpec``: Sets the default pod spec for all non-gpu tasks.
     See :ref:`custom-pod-specs` for details.
 
   - ``gpuPodSpec``: Sets the default pod spec for all ngpu tasks.
     See :ref:`custom-pod-specs` for details.
 
+  - ``cpuImage``: Sets the default docker image for all non-gpu tasks. If a docker image
+    is specified in the :ref:`experiment config <exp-environment-image>`
+    this default is overriden. Default to:
+    ``determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-0.5.0``.
+
+  - ``gpuImage``: Sets the default docker image for all gpu tasks. If a docker image
+    is specified in the :ref:`experiment config <exp-environment-image>`
+    this default is overriden. Default to:
```

Comment:
Maybe "Default to" -> "Defaults to"?

"Default to" sounds like a command to the reader.

(also above)